### PR TITLE
Fix group, org member tables in mobile view

### PR DESCRIPTION
**Pull request by wildcatzita**

### CODE REVIEW

**Comment by MrkGrgsn** [comment]
This new CSS will potentially alter the styling of more tables than you intend, since it will apply to the first 3 columns of any table that uses `col` elements. You need to use targeted CSS selectors, altering the template  to make if required (e.g., adding a class to the `col`'s or `table`)

**Comment by wildcatzita** [comment]
@MrkGrgsn, I've added class in templates instead col tag.

**Comment by smotornyuk** [comment]
@wildcatzita , i just noticed, that you'll get the same effect if you omit with attribute on col tags, even without adding anything to css. Could you check this and update you PR if i right?

**Comment by wildcatzita** [comment]
@smotornyuk, fixed. Can yo check please?

File: ckan/templates/group/members.html [hunk 1]
```diff
@@ -8,15 +8,12 @@
...
+  <table class="table table-header table-hover table-bordered group-org-members" id="member-table">
     <thead>
       <tr>
-        <th scope="col">{{ _('User') }}</th>
-        <th scope="col">{{ _('Role') }}</th>
-        <th scope="col"></th>
+        <th class="col-user">{{ _('User') }}</th>
+        <th class="col-role">{{ _('Role') }}</th>
+        <th class="col-actions"></th>
       </tr>
```
Comment (smotornyuk) [inline ckan/templates/group/members.html]: Are you still need these classes?

**Comment by smotornyuk** [comment]
I think it can be merged now

**Pull request merged**